lib/packwerk.rb

```diff
@@ -101,26 +96,10 @@ module Checkers
       extend ActiveSupport::Autoload
 
       autoload :DependencyChecker
-      autoload :PrivacyChecker
     end
   end
 
   private_constant :ReferenceChecking
-
-  class ApplicationValidator
-    extend ActiveSupport::Autoload
-
-    autoload :Helpers
-  end
 end
 
 require "packwerk/version"
-
-# Required to register the DefaultOffensesFormatter
-# We put this at the *end* of the file to specify all autoloads first
-require "packwerk/formatters/default_offenses_formatter"
-
-# Required to register the default DependencyChecker
-require "packwerk/reference_checking/checkers/dependency_checker"
```

Comment:
Do the autoloads take care of this registering? I recall us adding these because without a reference to them in the codebase that will cause zeitwerk to require them, these don't get properly registered.